nix/devcontainer/devcontainer.nix

```diff
@@ -35,7 +39,12 @@ let
     inherit name tag;
 
     contents = [
-      ./root
+      # See: https://github.com/NixOS/nixpkgs/issues/118722
+      (stdenv.mkDerivation {
```

Comment:
@gilligan any thoughts?

Reply:
I guess you can just delete this now.

Reply:
i don't think so because i'm keeping the ./root folder still because of the `skel` folder which has the default `.bashrc` for the plutus user.

Reply:
Oh right.